src/api/PDFPage.ts

```diff
@@ -1369,38 +1372,20 @@ export default class PDFPage {
     assertOrUndefined(options.y, 'options.y', ['number']);
     assertOrUndefined(options.xScale, 'options.xScale', ['number']);
     assertOrUndefined(options.yScale, 'options.yScale', ['number']);
+    assertOrUndefined(options.rotate, 'options.rotate', [[Object, 'Rotation']]);
     assertOrUndefined(options.color, 'options.color', [[Object, 'Color']]);
     assertRangeOrUndefined(options.opacity, 'opacity.opacity', 0, 1);
     assertOrUndefined(options.borderColor, 'options.borderColor', [
       [Object, 'Color'],
     ]);
-    assertRangeOrUndefined(
-      options.borderOpacity,
-      'options.borderOpacity',
-      0,
-      1,
-    );
-    assertOrUndefined(options.borderWidth, 'options.borderWidth', ['number']);
-    assertOrUndefined(options.borderDashArray, 'options.borderDashArray', [
-      Array,
-    ]);
-    assertOrUndefined(options.borderDashPhase, 'options.borderDashPhase', [
-      'number',
-    ]);
-    assertIsOneOfOrUndefined(
-      options.borderLineCap,
-      'options.borderLineCap',
-      LineCapStyle,
-    );
-    assertIsOneOfOrUndefined(options.blendMode, 'options.blendMode', BlendMode);
-
```

Comment:
Any reason for removing these assertions?

Reply:
I think it was when I switched to the "drawSvgPath" method inside the "drawEllipse", because "drawSvgPath" already had these assertions. But you're right, since we reverted that change, I'd need to revert this as well